cmd/importer/db.go

```diff
@@ -198,13 +198,6 @@ func genColumnData(table *table, column *column) (string, error) {
 		if isUnique {
 			data = float64(uniqInt64Value(column, 0, math.MaxInt64))
 		} else {
```

Comment:
Why remove the code block?

Reply:
Good catch, this code block is a possible bug.

Reply:
@jackysp Is this a kind of bug? Or just unused code?

Reply:
I think it is a bug, maybe it should be

```
if column.hist != nil {
...
} else {
...
}
```

Reply:
I think the code inside this package is not used for now. How about not changing them in this PR? Let's focused on solving golangci-lint?

Reply:
Should I keep it as it was? @zz-jason

Reply:
never mind